src/common/layout/Header.js

```diff
@@ -47,6 +47,7 @@ const loggedInStyle = css`
   .loggedIn {
     display: flex;
     justify-content: space-between;
+    flex: 1;
```

Comment:
I would rather rename the class - they still can decide to change styles for `Header` anytime, which could affect us.